docs/src/modules/components/AppFrame.js

```diff
@@ -197,6 +220,17 @@ class AppFrame extends React.Component {
             <div className={classes.root}>
               <NProgressBar />
               <CssBaseline />
+              <Button
```

Comment:
I don't think that we should be using a button look. The user might think he can tab on it, while he can't because it's implemented with a link. I would encourage the usage of `<Link />`.

Reply:
> I don't think that we should be using a button look

"if the user navigates very quickly using the Tab key, the link may be visible on the page for only a fraction of a second and may be overlooked. This can be partially addressed by ensuring that the "skip" link is visually distinctive at the top of the page."

https://webaim.org/techniques/skipnav/#invisible

The secondary button meets those criteria.

>  The user might think he can tab on it

❓

The user won't think anything - *unless* they tab onto it, because otherwise it's hidden.

Reply:
> The user might think he can tab on it

~tab~ space